runtime/ops/worker_host.rs

```diff
@@ -255,6 +255,13 @@ impl<'de> de::Visitor<'de> for ParseBooleanOrStringVec {
     formatter.write_str("a vector of strings or a boolean")
   }
 
+  fn visit_unit<E>(self) -> Result<UnaryPermissionBase, E>
+  where
+    E: de::Error,
+  {
+    self.visit_bool(false)
+  }
```

Comment:
A comment here explaining why unit is encoded as `false` would be helpful

Reply:
This is basically a means to map missing/undefined to `false`

I haven't looked into depth why this worked before with `serde_json`, but basically we implement `visit_unit` since we call `deserialize_any`, which in `serde_v8` will reflect the v8 value and for null/undefined call `visit_unit`.